src/skvhttpclient/src/skvhttp/client/SKVClient.h

```diff
@@ -148,7 +148,7 @@ class TxnHandle {
     boost::future<Response<dto::SKVRecord>> read(dto::SKVRecord record);
     boost::future<Response<>> write(dto::SKVRecord& record, bool erase=false,
                                        dto::ExistencePrecondition precondition=dto::ExistencePrecondition::None);
-    boost::future<Response<>> partialUpdate(dto::SKVRecord& record, std::vector<String> fieldNamesForUpdate);
+    boost::future<Response<>> partialUpdate(dto::SKVRecord& record, std::vector<uint32_t> fieldsForPartialUpdate, dto::ExistencePrecondition precondition=dto::ExistencePrecondition::None);
```

Comment:
A partialupdate implies dto::ExistencePrecondition::Exists, so we shouldn't let the user specify it here.

Reply:
Fixed